CONTRIBUTORS.md

```diff
@@ -0,0 +1,15 @@
+# Contributors
+
+## Project Lead
+
+* [Roberto Pastor](https://github.com/RobPasMue)
+
+## Individual Contributors
+
+* [Alex Kaszynski](https://github.com/akaszynski)
+* [Camille Latapie](https://github.com/clatapie)
+* [Jorge Martínez](https://github.com/jorgepiloto)
+* [Kathy Pipet](https://github.com/PipKat)
+* [Maxime Rey](https://github.com/MaxJPRey)
+* [Revathy Venugopal](https://github.com/Revathyvenugopal162)
+* [German Martinez Ayuso](https://github.com/germa89)
```

Comment:
```suggestion
# Contributors

## Project Lead

* [Camille Latapie](https://github.com/clatapie)
* [German Martinez Ayuso](https://github.com/germa89)
* [Jorge Martínez](https://github.com/jorgepiloto)
* [Maxime Rey](https://github.com/MaxJPRey)
* [Revathy Venugopal](https://github.com/Revathyvenugopal162)
* [Roberto Pastor](https://github.com/RobPasMue)

## Individual Contributors

* [Alex Kaszynski](https://github.com/akaszynski)
* [Kathy Pippert](https://github.com/PipKat)

```

I would add all of the @ansys/pyansys-core team members that have contributed

Reply:
As long as the bot is happy about it...

Reply:
Yep - we accept several project leads in the regex :)